Gemfile

```
@@ -1,6 +1,9 @@
 # frozen_string_literal: true
 
 source "https://rubygems.org"
+source 'https://gems.contribsys.com/' do
+  gem 'sidekiq-pro', '< 6'
+end
```

@cesarediaz for local development only, you will want to adjust this:
```suggestion
gem "sidekiq", "~> 6.0"
```

Also, I suggest you follow standard and use double quotes.

---

@knutsenm  I'm working in localhost with  the gem like 
```ruby
  group :development, :test do
    gem "sidekiq", "~> 7.1", ">= 7.1.2"
  end
```
but when pushing code I ommit it

this is what do you mean... avoid to work with a "pro" sidekiq gem on my localhost?

---

now I just will fix the single quote by replacing them for double ones on the sidekiq pro lines until you answer me

---

> this is what do you mean... avoid to work with a "pro" sidekiq gem on my localhost?

Yes, but stick with sidekiq 6